### PR TITLE
get device silently

### PR DESCRIPTION
**Pull request by AlvaroVega**
related with https://github.com/telefonicaid/iotagent-node-lib/pull/891

needs https://github.com/telefonicaid/iotagent-node-lib/pull/896

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Fix: Do not intercept error about DEVICE_NOT_FOUND in findOrCreate device (iotagent-node-lib#889)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
Fix: Do not intercept error about DEVICE_NOT_FOUND in findOrCreate device (iotagent-node-lib#889)
```

Similar fix needed in IOTA-UL code? Or it is alredy fixed there?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: No, only this agent re implement findOrCreate function of iotagent-node-lib

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

Travis relaunched, once that https://github.com/telefonicaid/iotagent-node-lib/pull/896 has been merged.

**Comment by AlvaroVega** [comment]
Still depends on PR https://github.com/telefonicaid/iotagent-node-lib/pull/899

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: twin PR https://github.com/telefonicaid/iotagent-ul/pull/439

**Pull request merged**